### PR TITLE
Combat timers for Beasts and Jaraxxus

### PR DESCRIPTION
**Pull request by ajames27**
Noticed the combat timers were off by a few seconds for the first two bosses.

For reference this is what the timers look like before the change

https://youtu.be/6KVnu6bHaKo?t=188

### CODE REVIEW

**Comment by MysticalOS** [comment]
i remember having to adjust these many times during wrath, cause they could in fact be shorter sometimes. they were variable, I wouldn't trust changing them without literally couple dozen videos showing that is not still the case with classic.

**Comment by ajames27** [comment]
Makes sense, would be nice to have two timers where one is a safe prepot timer and the other is a best guess for the median, but it's probably something for a weak aura.

**Comment by MysticalOS** [comment]
I would like a DBM timer feature eventually where I can have split bar or rather a line and stage 2 version of bar where I can more clearly show CD windows on things. like i fI now an ability is cast every 23-27 seconds, have a line in the bar at 23 and then from 23-37 when it passes that line shits to a diff color or something..Unfortunately @QartemisT has been super busy of late and most core work has been put on hold for now..

**Comment by MysticalOS** [comment]
I'll leave this open cause it is possible the min window is larger in classic, or more consistent, but I'd want to see more resets/data first to be sure before making this change

**Comment by ajames27** [comment]
Just as a note, the combat lock for beasts happens before the current timers have expired so there is no real risk of losing a pre-pot on that fight by changing the timer. You're in combat about 10 seconds before the boss becomes active.

**Review by MysticalOS — APPROVED** [review]

**Comment by MysticalOS** [comment]
I'm gonna go ahead and merge this, if the early start thing is still a thing, it's not that common. someone did 22 wipes on jaraxxus and said the timer was always later.

**Pull request merged**
